packages/plugins/i18n/admin/src/contentManagerHooks/editView.tsx

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable check-file/filename-naming-convention */
+import * as React from 'react';
+
+import { Flex, VisuallyHidden } from '@strapi/design-system';
+import { Earth, EarthStriked } from '@strapi/icons';
+import { EditFieldLayout, EditLayout } from '@strapi/strapi/admin';
+import { MessageDescriptor, useIntl } from 'react-intl';
+import styled from 'styled-components';
+
+import { getTranslation } from '../utils/getTranslation';
+
+interface MutateEditViewArgs {
+  layout: EditLayout;
+  query: object;
+}
+
+const mutateEditViewHook = ({ layout }: MutateEditViewArgs) => {
+  if (
+    'i18n' in layout.options &&
+    typeof layout.options.i18n === 'object' &&
```

Comment:
can i18n not be an object? :O

Reply:
layout.options only has the core opts and i18n is a plugin opt, so we have to assert it's what we think it is lol